examples/spring-boot/marketplace-app/src/main/resources/application.yml

```
@@ -0,0 +1,22 @@
+spring:
+  security:
+    oauth2:
+      client:
+        provider:
+          vendasta:
+            token-uri: 'https://sso-api-prod.apigateway.co/oauth2/token'
+            issuer-uri: 'https://iam-prod.apigateway.co'
+            authorization-uri: 'https://sso-api-prod.apigateway.co/oauth2/auth'
+            jwk-set-uri: 'https://iam-prod.apigateway.co/oauth2/v1/certs'
+            user-info-uri: 'https://sso-api-prod.apigateway.co/oauth2/user-info'
```

We can reference other application variables so that it is easier to switch between demo and prod.
```suggestion
            token-uri: 'https://sso-api-${apigateway.environment}.apigateway.co/oauth2/token'
            issuer-uri: 'https://iam-${apigateway.environment}.apigateway.co'
            authorization-uri: 'https://sso-api-${apigateway.environment}.apigateway.co/oauth2/auth'
            jwk-set-uri: 'https://iam-${apigateway.environment}.apigateway.co/oauth2/v1/certs'
            user-info-uri: 'https://sso-api-${apigateway.environment}.apigateway.co/oauth2/user-info'
```